src/convertToObject.js

```diff
@@ -7,6 +7,16 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stringArr = sourceString
+    .split(';')
+    .map((el) => el.split(':'))
+    .filter((el) => el.length > 1);
+
+  const styleObj = {};
+
+  stringArr.map((el) => (styleObj[el[0].trim()] = el[1].trim()));
```

Comment:
The `map` method is being used here incorrectly because it is intended to return a new array and not to perform side effects. In this case, you are using it to iterate over `stringArr` and modify `styleObj`. A more appropriate method to use here would be `forEach`, as it is intended for side effects and does not return a new array.